docs/CHANGELOG/changelog-0.9.x.md

```diff
@@ -1,79 +1,79 @@
+- Support added for Ubuntu 24.04 in Azure and Hetzner [#1401](https://github.com/berops/Claudie/pull/1401)
 
-- Each nodepool now has its own SSH keys, a change from the previous state where all nodepools shared the same SSH keys.. [#1442](https://github.com/berops/claudie/pull/1442)
+- Each nodepool now has its own SSH keys, a change from the previous state where all nodepools shared the same SSH keys.. [#1442](https://github.com/berops/Claudie/pull/1442)
 
-- Added support for pluggable external terraform files, breaking the dependency of updated terraform files on a new claudie version. [#1460](https://github.com/berops/claudie/pull/1460)
+- Added support for pluggable external terraform files, breaking the dependency of updated terraform files on a new Claudie version. [#1460](https://github.com/berops/Claudie/pull/1460)
 
-- With the support of external terraform templates, the ability to arbitrarily change the templates used by claudie was made available to the user. As a result, claudie has implemented a rolling update of the infrastructure in case a change in the terraform templates is detected, by gradually updating the build cluster one nodepool at a time. [#1525](https://github.com/berops/claudie/pull/1525)
+- With the support of external terraform templates, the ability to arbitrarily change the templates used by Claudie was made available to the user. As a result, Claudie has implemented a rolling update of the infrastructure in case a change in the terraform templates is detected, by gradually updating the build cluster one nodepool at a time. [#1525](https://github.com/berops/Claudie/pull/1525)
 
-- The Scheduler and Context-Box microservices were merged into a single service called Manager. This was done because these two services were tightly coupled, and parts of the context box service were causing state correctness issues within Claudie and needed to be fixed. [#1498](https://github.com/berops/claudie/pull/1498)
+- The Scheduler and Context-Box microservices were merged into a single service called Manager. This was done because these two services were tightly coupled, and parts of the context box service were causing state correctness issues within Claudie and needed to be fixed. [#1498](https://github.com/berops/Claudie/pull/1498)
 
-- Latest supported kubernetes version is now  v1.30.x [#1498](https://github.com/berops/claudie/pull/1501)
+- Latest supported kubernetes version is now  v1.30.x [#1498](https://github.com/berops/Claudie/pull/1501)
 
-- Logs in all microservices have been changed to always log what is being executed, rather than only when the LOG_LEVEL is set to debug. [#1507](https://github.com/berops/claudie/pull/1507)
+- Logs in all microservices have been changed to always log what is being executed, rather than only when the LOG_LEVEL is set to debug. [#1507](https://github.com/berops/Claudie/pull/1507)
```

Comment:
```suggestion
- Logs in all microservices have been changed to always log what is being executed, rather than only when the LOG_LEVEL is set to debug. [#1507](https://github.com/berops/claudie/pull/1507)
```